package golovatch.courses.exception.labs.lab_3_TWR_3;

/**
 * Created by Alex on 8/6/2016.
 */
public interface Three {
    public void call(AutoClosableFactory factoryA, AutoClosableFactory factoryB, AutoClosableFactory factoryC, TryBody tryBody) throws Throwable;
}
